website/screens/components/nav-tabs/specs/NavTabsSpecsPage.tsx

```diff
@@ -0,0 +1,279 @@
+import {
+  DxcBulletedList,
+  DxcFlex,
+  DxcTable,
+  DxcParagraph,
+} from "@dxc-technology/halstack-react";
+import DocFooter from "@/common/DocFooter";
+import Figure from "@/common/Figure";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import Image from "@/common/Image";
+import Code from "@/common/Code";
+import navTabsSpecs from "./images/navTabs_specs.png";
+import navTabsAnatomy from "./images/navTabs_anatomy.png";
+import navTabsStates from "./images/navTabs_states.png";
+
+const sections = [
+  {
+    title: "Specifications",
+    content: (
+      <Figure caption="Nav tabs design specifications">
+        <Image src={navTabsSpecs} alt="Nav tabs design specifications" />
+      </Figure>
+    ),
+  },
+  {
+    title: "States",
+    content: (
+      <>
+        <DxcParagraph>
+          Tabs can get different states based on user interaction. States:{" "}
```

Comment:
I think the word "States: " is a bit redundant here, I suggest two different options based on other components specs:

> "Tabs can get different states based on user interaction: **enabled**, **hover**, **focus**, **active** and **disabled**."

> "Tabs can get different states based on user interaction. These states are: **enabled**, **hover**, **focus**, **active** and **disabled**."

Reply:
Agree. Updated this also in the tabs specs.